### PR TITLE
Fix epoch dashboard error if websocket is not connected

### LINKED ISSUE #162: Nothing is displayed on epoch dashboard if websocket is not connected
image

Better show some hints on what is going on.

### PR DESCRIPTION
**Pull request by MonikaCat**
Fixes #162

### CODE REVIEW

File: components/Home/Epoch.tsx [hunk 1]
```diff
@@ -289,90 +274,84 @@ const Epoch = (): JSX.Element => {
...
-                                        className={classes.blockProposerName}>
-                                        {blockProposer.data.blockAdded.miner.name}
-                                    </Typography>
-                                ) : (
+                        <Grid item xs={12} className={classes.blockProposer}>
+                            {blockProposer?.data?.blockAdded?.miner?.name ||
+                            blockProposer?.data?.blockAdded?.miner?.address ? (
+                                <>
+                                    <img
+                                        src={`https://ui-avatars.com/api/?rounded=true&size=40&name=${
```
Comment (kwunyeung) [inline components/Home/Epoch.tsx:334]: How about we switch this to https://jdenticon.com/ ?

**Review by kwunyeung — CHANGES_REQUESTED** [review]
Please fix lint.

File: components/Home/Epoch.tsx (hunk 1 above)
Comment (MonikaCat) [reply components/Home/Epoch.tsx]: Ok, I added Avatar component and switched to using jdenticon library to generate icons :)

**Comment by MonikaCat** [comment]
> Please fix lint.

Lint and build pass with no errors

**Review by kwunyeung — APPROVED** [review]

**Pull request merged**